### PR TITLE
Add `getfield` instance method to NumPy Frontend

### PR DESCRIPTION
**Pull request by edith007**
Close https://github.com/unifyai/ivy/issues/14195

### CODE REVIEW

**Comment by edith007** [comment]
@karalleyna Let me know if there are any suggestions or changes required ? 🙇🏻

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

File: ivy/functional/frontends/numpy/ndarray/ndarray.py [hunk 1]
```diff
@@ -266,6 +266,9 @@ def flatten(self, order="C"):
         else:
             return np_frontend.ravel(self._ivy_array, order="C")
 
+    def getfield(self, dtype, offset=0):
+        return np_frontend.getfield(self._ivy_array, dtype, offset).astype(dtype)
```
Comment (hirwa-nshuti) [inline ivy/functional/frontends/numpy/ndarray/ndarray.py:270]: Hey this function isn't implemented on the numpy frontend
You'll have to implement it using the composition of ivy functions

**Review by hirwa-nshuti — CHANGES_REQUESTED** [review]
Hey @edith007 

Thanks for the PR😊
Tests are failing and a few changes are needed check the comments below.
Thanks🙂

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by ivy-seed** [comment]
This PR has been closed because it has been marked as stale for more than 7 days with no activity.

**Pull request closed without merging**